content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
+inform your student of your concerns, and ask the student why this is happening.
+Often students are blocked on a technical problem and do not communicate with their mentors.
+As a rule of thumb, there should be code pushed to Github almost every day.
+If not, let the Org Admins know as soon as possible.
+
+=== After the program
+
+Many students ask their mentors how they can continue to contribute after the program.
+
+Usually, you can invite the student to look at the bugs and features that have been captured
+in Jira during the coding phases for inspiration on what to do next.
+You can also invite students to apply next year as a student or even as a mentor.
+Students like to see their project continue, and becoming a mentor is a great way to make that happen.
+
+Google organizes a mentor summit a few months after the program has ended.
+Each year, the Jenkins Org Admins select 2 mentors who get to go to that summit
+(travel costs and accommodations are paid for by Google).
+Mentors agree that this is a fantastic event!
+
+Some mentors travel to the Jenkins World conference and get to meet students and other mentors.
+This is definitely a conference worth attending for mentors and students alike.
+
+Here are some posts by mentors from past years conferences:
+
+* link:/blog/2019/11/2019-11-17-gsoc-report/[GSoC 2019 Report], multiple authors
+* link:http://jeff-pearce.com/google-summer-of-code-with-the-jenkins-project/[Google Summer of Code with the Jenkins Project], by Jeff Pearce
+* link:/blog/2018/11/13/martinda-gsoc-mentor-summit-experience/[Google Summer of Code Mentor Summit 2018], by Martin d'Anjou
+* link:/blog/2019/10/08/google-summer-of-code-mentor-and-org-admin-perspective/[Google Summerof Code Mentor and Org Admin Perspective], by Marky Jackson
+* link:/blog/2019/08/25/jenkinsworld-contrib-summit-ask-the-expert-booth/[Jenkins World Contributor Summit and Ask the Exper booth], by Marky Jackson
+* link:/blog/2018/10/14/gsoc2018-results/[Jenkins in Google summer of Code 2018 Results] by Oleg Nenashev
 
+== Regarding mentor time off
 
-=== Expectations from mentors per phase
+The GSoC program lasts several months.
+We know people go on vacation and need to take time off from their regular day job.
+We are flexible.
+This is one of the reason why we also assign at least two mentors per student.
+Make sure you communicate your availability to your student, your co-mentors, and the org admins.
 
-This section is under construction.
+If you must withdraw from the program completely, let the Jenkins Org Admins know as soon as possible.
 
-=== Conflict of interest prevention
+== Conflict of interest prevention
 
-We will appreciate mentorship provided by any Jenkins contributor.
+We appreciate mentorship provided by any Jenkins contributor.
 On the other hand, we want to avoid any conflicts with GSoC rules and spirit.
 We also want to avoid conflicts of interest between all sides.
```

Comment:
I'm a little at a loss as to what sorts of things would constitute a conflict of interest. Is it possible to give some examples?

Reply:
There are examples below in the same doc. If you have a case that is not covered, we can address it in another PR.